Parsers/ASimAuditEvent/Parsers/ASimAuditEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,183 @@
+            | parse description_s with "Policy " temp_policyname3: string " (ID: " temp_policyid3 ") " temp_type3: string
+            | extend
+                Object = coalesce(temp_policyname1, temp_policyname2, temp_policyname3),
+                ObjectType = "Policy Rule",
+                temp_type = replace_regex(coalesce(temp_type1, temp_type2, temp_type3), @'[is,was]* (\S+)', @'\1'),
+                OldValue = temp_policyname2,
+                AdditionalFields = bag_pack("policy id", coalesce(temp_policyid1, temp_policyid2, temp_policyid3))
+            | lookup EventTypeLookup on temp_type
+            | extend
+                Operation = iff(isnotempty(temp_type2), strcat("Policy ", Object, " ", temp_type, " and renamed to ", NewValue), strcat("Policy ", Object, " ", temp_type));
+        let Changed = allData
+            | where description_s startswith "Changed"
+            | parse description_s with temp_operation: string " to " NewValue: string " (ID: " temp_policyid ")" * "device(s): " temp_deviceid: string 
+            | extend
+                EventType = "Set",
+                Operation = strcat(temp_operation, " to ", NewValue, " for ID: ", temp_policyid),
+                Object = strcat("Policy ", temp_policyid),
+                ObjectType = "Policy Rule",
+                AdditionalFields = bag_pack("devices", temp_deviceid),
+                TargetDvcId = temp_deviceid;
+        let ParamsUpdated = allData
+            | where description_s startswith "Parameters updated"
+            | parse description_s with "Parameters updated for " temp_config1: string " (ID: " temp_configid1: string ") for policy " temp_policyname1: string " (ID: " temp_policyid1: string ")" temp_restmessage1: string
+            | parse description_s with "Parameters updated for " temp_config2: string " (ID: " temp_configid2: string ") for policy with ID " temp_policyid2: string
+            | extend
+                temp_operation = coalesce(temp_config1, temp_config2),
+                temp_configid = coalesce(temp_configid1, temp_configid2)
+            | extend
+                EventType = "Set", 
+                Operation = strcat("Parameters updated for ", temp_operation, " for policy ", temp_policyname1, tostring(split(temp_policyid2, "{")[0])),
+                Object = strcat("Policy ", coalesce(temp_policyname1, temp_policyid2)),
+                ObjectType = "Policy Rule",
+                AdditionalFields = bag_pack("config id", temp_configid);
+        let Reputation = allData
+            | where description_s has_cs "Reputation"
+            | parse description_s with "User " * " " temp_type1: string " Reputation" * " for Organization ID " temp_orgid1: string " of type " temp_reptype1: string " to " temp_list1: string " with content: " temp_content1: string " | " temp_restmessage1: string
```

Comment:
we are parsing temp_orgid1 but not using? Please map this with TargetDvcScopeId

Reply:
orgName_s represents the name of the organization in which a user exists who has performed this audit activity in VMware Carbon Black Cloud. Hence, we can map it with ActorScopeId instead of TaretDvcScopeId.

Reply:
@niralishah-crest sounds good

Reply:
saw you already did this mapping.